conceptnet5/vectors/evaluation/bias.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+import pandas as pd
+import scipy
+
+from conceptnet5.vectors import standardized_uri, normalize_vec
+from conceptnet5.vectors.transforms import (
+    l2_normalize_rows, subtract_mean_vector
+)
+from conceptnet5.vectors.debias import (
+    FEMALE_WORDS, MALE_WORDS, PEOPLE_BY_BELIEF, PEOPLE_BY_ETHNICITY,
+    CULTURE_PREJUDICES, get_category_axis, get_vocabulary_vectors
```

Comment:
I think you may be looking at debias.py, not bias.py. With this comment, I was mostly wondering if this was intended that you don't measure `CULTURE_PREJUDICES`.